.nuspec/Microsoft.Maui.Resizetizer.targets

```diff
@@ -290,6 +290,16 @@
             <MauiImage Include="@(MauiIcon)" IsAppIcon="True" />
         </ItemGroup>
 
+        <!-- Add a hash to the splash screen so we can use it later -->
+        <GetFileHash Files="@(MauiSplashScreen)" HashEncoding="base64">
+            <Output TaskParameter="Items" ItemName="_MauiSplashScreensWithHashes" />
+        </GetFileHash>
+        <ItemGroup>
+            <_MauiSplashScreensWithHashes
+                Update="@(_MauiSplashScreensWithHashes)"
+                FileHash="$([MSBuild]::ValueOrDefault('%(FileHash)', '').Replace('+','').Replace('/','').Replace('=',''))" />
```

Comment:
If you let it do `hex`, can you avoid the several `Replace()` calls here?

Reply:
I was using hex, but it ended up in a very long string.